<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addcomstudlg</class>
 <widget class="QDialog" name="addcomstudlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>197</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>学生学号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="stuIdLineEdit">
        <property name="placeholderText">
         <string>此项必填</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>竞赛代码：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="comIdLineEdit">
        <property name="placeholderText">
         <string>此项必填</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>获奖等级：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="gradeComboBox">
        <item>
         <property name="text">
          <string>一等奖</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>二等奖</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>三等奖</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>优秀奖</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>获奖时间：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
